lib/main/STM32H7/Middlewares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc.c

```diff
@@ -264,9 +264,7 @@ __ALIGN_BEGIN static uint8_t USBD_MSC_DeviceQualifierDesc[USB_LEN_DEV_QUALIFIER_
   * @}
   */
 
-
-// Buffer must be aligned for DMAs
-static DMA_RW_AXI USBD_MSC_BOT_HandleTypeDef ClassData __attribute__((aligned(32)));
+static DMA_DATA_ZERO_INIT USBD_MSC_BOT_HandleTypeDef ClassData;
```

Comment:
This should go into a separate pull request so it can easily be re-applied after library updates.

Reply:
This ties into the other changes made to tidy up the use of memory suitable for DMA. I'll split this one out though as it's in the library.

Reply:
See https://github.com/betaflight/betaflight/pull/10747